src/main/java/org/proshin/finapi/mock/in/CategorizationParameter.java

```diff
@@ -33,6 +33,10 @@ public CategorizationParameter(final String transaction, final Type type, final
         );
     }
 
+    public CategorizationParameter(final Supplier<JSONObject> origin) {
+        this.origin = origin;
+    }
+
```

Comment:
A primary ctor should always be at the end of the list of ctors.

Reply:
But it is the last ctor in the file.
Unless you consider those methods "with" also ctors?